### DISCUSSION TITLE
Sim number security code

### DISCUSSION THREAD

**Discussion by jelliecasas** [community/community#30876] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to remove or deletes My old number 09303454049 and i want to change my new number now 09277091016

**Comment by airtower-luna** [comment, marked as answer]
You just posted your phone number on a public forum, I doubt that's what you wanted. :warning: 

If you want to change the number for SMS as second factor for authentication, you need to do it in your account settings, see https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication.
